flask/todo_app/README.rst

```diff
@@ -0,0 +1,31 @@
+Flask Todo App
+##############
+
+Quick Start
+===========
+
+Set environment vars for the Flask application::
```

Comment:
These instructions are for a local development setup that just uses Python and Flask.
Might be helpful on it's own but not as relevant with the code mounted in a docker container. Open to adding more context or removing.